pkgs/tools/cd-dvd/cdimgtools/nrgtool_fix_my.patch

```diff
@@ -0,0 +1,12 @@
+Fixes an error in nrgtool.
```

Comment:
It's usually nicer if you want a comment to just commit this into to the repo and then use `git format-patch`

##### This is just a comment, you don't need to do this

Reply:
What do you mean; what should I commit into which repo? :-)

Reply:
It's fine like this, and I meant commit this change into a local checkout of `cdimgtools` repo with this as the commit msg, and then use `git format-patch` to produce patches for the commits.

Reply:
Got it :-) Thanks for the info! :-)